website_docs/manual_instrumentation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Ruby Manual Instrumentation"
+weight: 24
+description: >
+  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Ruby_SDK.svg"></img>
+  A language-specific implementation of OpenTelemetry in Ruby.
+---
+
+## Adding Manual Instrumentation
+
+Auto-instrumentation is the easiest way to get started with instrumenting your code, but in order to get the most insight into your system, you should add manual instrumentation where appropriate.
+To do this, use the OpenTelemetry SDK to access the currently executing span and add attributes to it, and/or to create new spans.
+
+### Adding Context to Exisiting Spans
+
+It's often beneficial to add context to a currently executing span in a trace.
+For example, you may have an application or service that handles extended warranties, and you want to associate it with the span when querying your tracing datastore.
+In order to do this, get the current span from the context and set an attribute with your application's domain specific data:
+
+```ruby
+def track_extended_warranty(extended_warranty)
+  current_span = OpenTelemetry::Trace.current_span
+  current_span.add_attributes({
+    "com.extended_warranty.id" => extended_warranty.id,
+    "com.extended_warranty.timestamp" => extended_warranty.timestamp
+  })
+end
+```
+
+### Creating New Spans
+
+Auto-instrumentation can show the shape of requests to your system, but only you know the really important parts.
+In order to get the full picture of what's happening, you will have to add manual instrumentation and create some custom spans.
+To do this, grab the tracer from the OpenTelemetry API and generate a span:
+
+```ruby
+# ...
+
+def search_by(query)
+  tracer = OpenTelemetry.tracer_provider.tracer('my-tracer')
+  tracer.in_span("search_by") do |span|
+    # ... expensive query
+  end
+end
+```
+
+The `in_span` convience method is unique to Ruby implementation, which reduces some of the boilerplate code that you would have to otherwise write yourself:
```

Comment:
```suggestion
The `in_span` convenience method is unique to Ruby implementation, which reduces some of the boilerplate code that you would have to otherwise write yourself:
```